src/main/fc/parameter_names.h

```diff
@@ -117,3 +117,46 @@
 #define PARAM_NAME_RPM_FILTER_MIN_HZ "rpm_filter_min_hz"
 #define PARAM_NAME_RPM_FILTER_FADE_RANGE_HZ "rpm_filter_fade_range_hz"
 #define PARAM_NAME_RPM_FILTER_LPF_HZ "rpm_filter_lpf_hz"
+
+#ifdef USE_GPS
+#define PARAM_NAME_GPS_PROVIDER "gps_provider"
+#define PARAM_NAME_GPS_SBAS_MODE "gps_sbas_mode"
+#define PARAM_NAME_GPS_SBAS_INTEGRITY "gps_sbas_integrity"
+#define PARAM_NAME_GPS_AUTO_CONFIG "gps_auto_config"
+#define PARAM_NAME_GPS_AUTO_BAUD "gps_auto_baud"
+#define PARAM_NAME_GPS_UBLOX_USE_GALILEO "gps_ublox_use_galileo"
+#define PARAM_NAME_GPS_UBLOX_MODE "gps_ublox_mode"
+#define PARAM_NAME_GPS_SET_HOME_POINT_ONCE "gps_set_home_point_once"
+#define PARAM_NAME_GPS_USE_3D_SPEED "gps_use_3d_speed"
+
+#ifdef USE_GPS_RESCUE
+#define PARAM_NAME_GPS_RESCUE_ANGLE "gps_rescue_angle"
+#define PARAM_NAME_GPS_RESCUE_ALT_BUFFER "gps_rescue_alt_buffer"
+#define PARAM_NAME_GPS_RESCUE_INITIAL_ALT "gps_rescue_initial_alt"
+#define PARAM_NAME_GPS_RESCUE_DESCENT_DIST "gps_rescue_descent_dist"
+#define PARAM_NAME_GPS_RESCUE_LANDING_ALT "gps_rescue_landing_alt"
+#define PARAM_NAME_GPS_RESCUE_LANDING_DIST "gps_rescue_landing_dist"
+#define PARAM_NAME_GPS_RESCUE_GROUND_SPEED "gps_rescue_ground_speed"
+#define PARAM_NAME_GPS_RESCUE_THROTTLE_P "gps_rescue_throttle_p"
+#define PARAM_NAME_GPS_RESCUE_THROTTLE_I "gps_rescue_throttle_i"
+#define PARAM_NAME_GPS_RESCUE_THROTTLE_D "gps_rescue_throttle_d"
+#define PARAM_NAME_GPS_RESCUE_VELOCITY_P "gps_rescue_velocity_p"
+#define PARAM_NAME_GPS_RESCUE_VELOCITY_I "gps_rescue_velocity_i"
+#define PARAM_NAME_GPS_RESCUE_VELOCITY_D "gps_rescue_velocity_d"
+#define PARAM_NAME_GPS_RESCUE_YAW_P "gps_rescue_yaw_p"
+#define PARAM_NAME_GPS_RESCUE_ROLL_MIX "gps_rescue_roll_mix"
+#define PARAM_NAME_GPS_RESCUE_THROTTLE_MIN "gps_rescue_throttle_min"
+#define PARAM_NAME_GPS_RESCUE_THROTTLE_MAX "gps_rescue_throttle_max"
+#define PARAM_NAME_GPS_RESCUE_ASCEND_RATE "gps_rescue_ascend_rate"
+#define PARAM_NAME_GPS_RESCUE_DESCEND_RATE "gps_rescue_descend_rate"
+#define PARAM_NAME_GPS_RESCUE_THROTTLE_HOVER "gps_rescue_throttle_hover"
+#define PARAM_NAME_GPS_RESCUE_SANITY_CHECKS "gps_rescue_sanity_checks"
+#define PARAM_NAME_GPS_RESCUE_MIN_SATS "gps_rescue_min_sats"
+#define PARAM_NAME_GPS_RESCUE_MIN_DTH "gps_rescue_min_dth"
+#define PARAM_NAME_GPS_RESCUE_ALLOW_ARMING_WITHOUT_FIX "gps_rescue_allow_arming_without_fix"
+#define PARAM_NAME_GPS_RESCUE_ALT_MODE "gps_rescue_alt_mode"
+#ifdef USE_MAG
+#define PARAM_NAME_GPS_RESCUE_USE_MAG "gps_rescue_use_mag"
+#endif
+#endif
+#endif
```

Comment:
```suggestion
#endif

```

Reply:
Thank you!

Reply:
I was suggesting to add a missing line break here :)

Reply:
```suggestion
#define PARAM_NAME_GPS_RESCUE_ALLOW_ARMING_WITHOUT_FIX "gps_rescue_allow_arming_without_fix"
#define PARAM_NAME_GPS_RESCUE_ALT_MODE "gps_rescue_alt_mode"

#ifdef USE_MAG
#define PARAM_NAME_GPS_RESCUE_USE_MAG "gps_rescue_use_mag"
#endif

#endif
#endif

```

Reply:
Ah... OK thanks, will do